libs/map/src/components/action/layer-action/select-action/multiselect-actions/vl-map-multiselect-actions.cy.ts

```diff
@@ -0,0 +1,181 @@
+import { html } from 'lit';
+import { registerWebComponents } from '@domg-wc/common-utilities';
+import { VlMap } from '../../../../../vl-map';
+import { VlMapBaseLayerGRBGray } from '../../../../baselayer/vl-map-base-layer-grb-gray/vl-map-base-layer-grb-gray';
+import { VlMapFeaturesLayer } from '../../../../layer/vector-layer/vl-map-features-layer/vl-map-features-layer';
+import { VlMapMultiselectActions } from './vl-map-multiselect-actions';
+
+registerWebComponents([VlMap, VlMapBaseLayerGRBGray, VlMapFeaturesLayer, VlMapMultiselectActions]);
+
+const featuresLayer1 = {
+    type: 'Feature',
+    id: 1,
+    geometry: {
+        type: 'Polygon',
+        coordinates: [
+            [
+                [100000, 150000],
+                [200000, 150000],
+                [200000, 250000],
+                [100000, 250000],
+                [100000, 150000],
+            ],
+        ],
+    },
+};
+
+const featuresLayer2 = {
+    type: 'FeatureCollection',
+    features: [
+        {
+            type: 'Feature',
+            id: 2,
+            geometry: {
+                type: 'Polygon',
+                coordinates: [
+                    [
+                        [100000, 140000],
+                        [190000, 140000],
+                        [190000, 200000],
+                        [100000, 200000],
+                        [100000, 140000],
+                    ],
+                ],
+            },
+        },
+    ],
+};
+
+const layers = ['layer-1', 'layer-2'];
+
+describe('component vl-map-multiselect-actions', () => {
+    beforeEach(() => {
+        cy.mount(html`
+            <vl-map>
+                <vl-map-baselayer-grb-gray></vl-map-baselayer-grb-gray>
+                <vl-map-features-layer data-vl-name="layer-1" .features=${featuresLayer1}>
+                    <vl-map-layer-style data-vl-border-size="2"></vl-map-layer-style>
+                </vl-map-features-layer>
+                <vl-map-features-layer .features=${featuresLayer2} data-vl-name="layer-2">
+                    <vl-map-layer-style data-vl-border-size="2"></vl-map-layer-style
+                ></vl-map-features-layer>
+                <vl-map-multiselect-actions .active=${true} .layers=${layers} ?data-vl-default-active=${true}>
+                </vl-map-multiselect-actions>
+            </vl-map>
+        `);
+    });
+
+    it('should mount', () => {
+        cy.get('vl-map-multiselect-actions');
+    });
+
+    it('should be accessible', () => {
+        cy.injectAxe();
+
+        cy.checkA11y('vl-map-multiselect-actions');
+    });
+
+    it('should fire the onSelect event', () => {
+        cy.window().then((win) => {
+            // @ts-ignore
```

Comment:
Kan je als je een ts-ignore plaatst een klein woordje uitleg erbij zetten waarom dit nodig is?

Reply:
Ik heb het gedeclared zodat @ts-ignore niet meer nodig is.